doc/whatsnew.rst

```diff
@@ -16,6 +18,7 @@ v2024.8.6
 - Add :doc:`/project/edits` project code and documentation (:pull:`204`).
 - Reduce log verbosity of :func:`.apply_spec` (:pull:`202`).
 - Fix and update :doc:`/api/tools-costs` (:pull:`186`, :pull:`187`, :pull:`190`, :pull:`195`).
+- Made fixes and updates to :doc:`/api/tools-costs` (:pull:`186`, :pull:`187`, :pull:`190`, :pull:`195`, :pull:`206`).
```

Comment:
Why do we get this new line instead of expanding the one above it (using "Fix und update ...")?

Reply:
Thanks for the catch :) Think I made an error within doc/whatsnew along the way by adding this PR #206 to the v2024.8.6 release instead of under "Next release". I deleted this entry under v2024.8.6, but this PR is documented under "Next release".

Reply:
We also have this line: https://github.com/iiasa/message-ix-models/blob/32cb2fd884afec2cf95878d9051cf4b33dd5af2c/.gitattributes#L1-L2 —but it is still incumbent on PR authors to keep the entries in the right order.

Reply:
Thanks all - after a few mistakes I think I sorted the `doc/whatsnew` issues. Took a few commits to get there but hope that's fine.

For the subsequent PRs #221, #227, and #222, I rebased and dropped all the other commits related to doc/whatsnew because they were creating some clutter. Then just did one final edit to doc/whatsnew for each PR.